src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -50,6 +59,12 @@ public function __construct($defaultContext = [], \DateTimeZone $timezone = null
             $defaultContext[self::TIMEZONE_KEY] = $timezone;
         }
 
+        if (!isset($defaultContext[self::TIMEZONE_KEY]) && null !== $timezone) {
+            @trigger_error('Passing configuration options directly to the constructor is deprecated since Symfony 4.2, use the default context instead.', E_USER_DEPRECATED);
```

Comment:
The message looks wrong to me?

Reply:
This message was not set by this PR, but is existent within accepted code.  Perhaps @nicolas-grekas would be best suited to change this message.

Reply:
Altered to show which configuration option had tripped the issue.